docs/content/users/install/ddev-installation.md

```diff
@@ -43,22 +43,18 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
 
     ### Debian/Ubuntu
 
-    DDEV’s Debian and RPM packages work with `apt` and `yum` repositories and most variants that use them, including Windows WSL2:
+    DDEV’s Debian and RPM packages work with `apt` and `yum` repositories and most variants that use them, including Windows WSL2. Make sure you have cURL installed:
 
     ```bash
     # Add DDEV’s GPG key to your keyring
-    sudo sh -c 'echo ""'
-    sudo apt-get update && sudo apt-get install -y curl
     sudo install -m 0755 -d /etc/apt/keyrings
     curl -fsSL https://pkg.ddev.com/apt/gpg.key | gpg --dearmor | sudo tee /etc/apt/keyrings/ddev.gpg > /dev/null
     sudo chmod a+r /etc/apt/keyrings/ddev.gpg
 
     # Add DDEV releases to your package repository
-    sudo sh -c 'echo ""'
```

Comment:
The purpose of these is to make sure that people have entered the sudo password. Otherwise, what happens is they paste the whole thing, the next line is taken as the password, and the installation fails.

We're open to a better way to accomplish this. It's very dependent on sudo configuration, but with the default configuration, people need to put a password in once for this set of instructions.